harness/decimalToHexString.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2017 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+function decimalToHexString(n) {
+  var hex = "0123456789ABCDEF";
+  n >>>= 0;
```

Comment:
This is actually the first time I used this assignment operator. Hang on while I read the specs, mdn, etc. :)